internal-packages/hub-db/src/client.ts

```diff
@@ -0,0 +1,64 @@
+import { PrismaClient } from "../generated/index.js";
+
+/**
+ * This file is contained in this package instead of apps/hub, because sometimes it can be useful to access the Prisma client from other apps or CLI scripts.
+ *
+ * Note that CLI scripts usefulness is limited by the fact that they don't have access to the Next.js runtime environment, e.g. auth context.
```

Comment:
My initial approach in this PR was to have evals code in a separate CLI-only `internal-packages/evals` package.

Then I realized that I need to share the data-loading code (selecting things directly with Prisma is more verbose and error-prone), so this comment became less relevant.

But I imagine that other apps like `website` or `metaforecast` might still need to use Hub's prisma code for something in the future.